Tribes/src/app/dashboard/dashboard.component.ts

```diff
@@ -53,5 +53,4 @@ export class DashboardComponent implements OnInit {
     .subscribe(response => this.numberOfBarracks = response.buildingDTOS
       .filter(building => building.buldingTypeENUM === 'BARRACK').length);
   }
-
 }
```

Comment:
When you do these formatting things, it increases the changes of conflicts of unrelated changes. 
The best practice usually is to have them in separate "code quality cleaning" tickets & PRs.